flake.nix

```diff
@@ -0,0 +1,33 @@
+{
+  description = "Nix-enabled environment for your Android device";
+
+  inputs = {
+    nixpkgs.url = "nixpkgs/nixos-unstable";
```

Comment:
You can omit this line. nixpkgs is a default input and the channel is irrelevant here